diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/IdentifierNaming.kt

```diff
@@ -162,6 +166,11 @@ class IdentifierNaming(private val configRules: List<RulesConfig>) : Rule("ident
                     // variable name should be in camel case. The only exception is a list of industry standard variables like i, j, k.
                     VARIABLE_NAME_INCORRECT_FORMAT.warnAndFix(configRules, emitWarn, isFixMode, variableName.text, variableName.startOffset, node) {
                         // FixMe: cover fixes with tests
+                        variableName
+                            .parent({it.elementType == FILE})
+                            ?.findAllVariablesWithUsages { it.name == variableName.text && (it.isLocal || it.isPrivate()) }
```

Comment:
I think, the `variableName` node should be local or private, what's the point in checking it's usages?